resources/views/vendor/apidoc/partials/info.blade.php

```diff
@@ -154,6 +158,60 @@
 refresh_token | string | The refresh token.
 
 
+## Client Credentials Grant
+
+The client credential flow provide a way for developers to get access tokens without user permissions; as such, these tokens are considered as guest users.
```

Comment:
"getting access tokens without user permissions" sounds a bit dangerous :upside_down_face:  "with no" instead of "without" perhaps? Or some other better wordings.

Reply:
...those both mean the same thing 🤔